corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -798,41 +803,6 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
     Group.prototype = Object.create(Container.prototype);
     Group.prototype.constructor = Container;
 
-    /**
-     * Represents a repeat group. A repeat only has Group objects as children, which are contained
-     * within a GroupedElementTileRow. Each child Group contains GroupedElementTileRow
-     * objects which contains the child questions to be rendered
-     * @param {Object} json - The JSON returned from touchforms to represent a Form
-     * @param {Object} parent - The object's parent. Either a Form, Group, or Repeat.
-     */
-    function Repeat(json, parent) {
-        var self = this;
-        self.parent = parent;
-
-        Container.call(self, json);
-
-        self.rel_ix = ko.observable(relativeIndex(self.ix()));
-        if (_.has(json, 'domain_meta') && _.has(json, 'style')) {
-            self.domain_meta = parseMeta(json.datatype, json.style);
-        }
-        self.templateType = 'repeat';
-        self.ixInfo = function (o) {
-            var fullIx = getIx(o);
-            return o.rel_ix + (o.isRepetition ? '(' + o.uuid + ')' : '') + (o.rel_ix !== fullIx ? ' :: ' + fullIx : '');
```

Comment:
There's similar logic in `Question` referencing `isRepetition`, is that intentional?
https://github.com/dimagi/commcare-hq/blob/e5ec4a4ef786e2efc2fc93f6c16e1c42389b76ee/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L933

Reply:
Need to look into that. 🤔